test/client/fixtures/sandboxes/cookie-test.js

```diff
@@ -84,7 +84,7 @@ test('get/set', function () {
 asyncTest('path validation', function () {
     var iframe = document.createElement('iframe');
 
-    iframe.src = UrlUtil.getProxyUrl('/data/cookie-sandbox/validation.html');
+    iframe.src = UrlUtil.getProxyUrl(window.QUnitGlobals.getResourceUrl('../../data/cookie-sandbox/validation.html', 'cookie-sandbox/validation.html'));
```

Comment:
see [here](https://github.com/AlexanderMoskovkin/qunit-harness#get-test-resource). It's neccessary for the cookies test. We need an iframe with url like `.../cookie-sandbox/iframe.html`. For me the best way is use:
`window.QUnitGlobals.getResourceUrl(<fileName>, 'cookie-sandbox/iframe.html')`. With the second parameter we set the resource url